website/docs/api/plugins/plugin-content-blog.md

```diff
@@ -186,6 +186,8 @@ Accepted fields:
 | `tags` | `Tag[]` | `undefined` | A list of strings or objects of two string fields `label` and `permalink` to tag to your post. |
 | `draft` | `boolean` | `false` | A boolean flag to indicate that the blog post is work-in-progress and therefore should not be published yet. However, draft blog posts will be displayed during development. |
 | `hide_table_of_contents` | `boolean` | `false` | Whether to hide the table of contents to the right. |
+| `toc_max_heading_level` | `number` | `4` | The max heading level shown in the table of contents. Must be between 2 and 6. |
+| `toc_min_heading_level` | `number` | `2` | The minimum heading level shown in the table of contents. Must be between 2 and 6 and lower or equal to the max value. |
```

Comment:
Honestly, don't really understand why we need option to set min level, it seems bit redundant. Usually people would need to specify max depth, or not?

Reply:
I took that from @Josh-Cena's review comment https://github.com/facebook/docusaurus/pull/5578#discussion_r710573699.

I'm personally a -0 on it (see https://github.com/facebook/docusaurus/pull/5578#discussion_r710572371), but wanted to write the implementation so that we could have it and just removed it if it was deemed unnecessary in the API.